CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## 0.17.18 (Unreleased)
 
+- update CONTRIBUTING.md to include `brew install go`
```

Comment:
We can remove this - since this is not part of the release - just part of the internal project.  I'll add the label so this doesn't get flagged as needing a `CHANGELOG` entry.